src/shared/components/image-button/ImageButton.tsx

```diff
@@ -24,52 +28,57 @@ import Tooltip from 'src/shared/components/tooltip/Tooltip';
 
 import { Status } from 'src/shared/types/status';
 
-import imageButtonStyles from './ImageButton.scss';
+import styles from './ImageButton.scss';
 
 export type ImageButtonStatus =
   | Status.DEFAULT
   | Status.SELECTED
   | Status.UNSELECTED
   | Status.DISABLED;
 
-export type Props = {
-  status: ImageButtonStatus;
-  description: string;
+type ChildlessButtonProps = Omit<
```

Comment:
why do we need this?

Reply:
we are not passing children through

Reply:
Yes; I also removed the `disabled` property; so that the only way to influence whether the ImageButton is disabled is to pass appropriate props to it.